### DISCUSSION TITLE
How do i customize the lualine shapes without overriding all the lazyvim defaults?

### DISCUSSION THREAD

**Discussion by thadkrugman** [LazyVim/LazyVim#3616] [Q&A] [answered]
![Screenshot 2024-06-12 at 3 52 04 PM](https://github.com/LazyVim/LazyVim/assets/63556794/fdb0dd22-b056-4dbe-8edb-364a1c1845b5)

I want lualine to use the rounded shapes like I'm using with p10k and tmux. I couldn't get it to work without overriding all the default settings from lazy.

Thanks in advance for any help.

**Comment by Aman9das** [comment]
lang:lua lualine return search like this for example configs

Mine is here: https://github.com/Aman9das/lazy-nvim-dotfiles/blob/main/lua/plugins/ui.lua

**Comment by rubiin** [comment]
@thadkrugman  try this
```lua
  {
    "nvim-lualine/lualine.nvim",
    opts = {
      options = {
        component_separators = "",
		section_separators = { left = "", right = "" }
      }
	}
}


```

**Reply by rubiin** [reply, marked as answer]
![image](https://github.com/LazyVim/LazyVim/assets/8222059/08ac9dad-35d6-4117-ba90-c3bcf4af5487)

**Reply by thadkrugman** [reply]
Thank you! 

This is what I ended up with.

```
return {
    {
        "nvim-lualine/lualine.nvim",
        opts = {
            options = {
                component_separators = "",
                section_separators = { left = "", right = "" },
            },
            sections = {
                lualine_a = { { "mode", separator = { left = "" }, right_padding = 2 } },
                lualine_y = {
                    { "progress", separator = " ", padding = { left = 1, right = 0 } },
                    { "location", padding = { left = 0, right = 1 } },
                },
                lualine_z = {
                    {
                        function()
                            return " " .. os.date("%R")
                        end,
                        separator = { right = "" },
                    },
                },
            },
        },
    },
}
```
![Screenshot 2024-06-14 at 3 01 03 PM](https://github.com/LazyVim/LazyVim/assets/63556794/7453110d-3e8e-445d-a0c3-6fc15c2604f9)
